### PR TITLE
doc: initial draft of 0.6 release notes

### PR DESCRIPTION
**Pull request by dbkinder**
Initial draft of release notes provided by Nanlin.

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

**Comment by hwang37** [comment]
Only one comment:
Should we change "A 101 blog to introduce Trusty on ACRN" title to "An 101 introduction for Trusty and Security Services in ACRN"?

**Comment by hwang37** [comment]
@wenlingz  to check the build CI failure

**Comment by wenlingz** [comment]
@hwang37 OK

**Comment by wenlingz** [comment]
@dbkinder build fail, because of below error,
********************************************************
New errors/warnings found, please fix them:
==============================================

/workspace/acrn-hypervisor/doc/release_notes_0.6.rst:111: WARNING: Inline emphasis start-string without end-string.
/workspace/acrn-hypervisor/doc/release_notes_0.6.rst:47: WARNING: undefined label: rt_linux (if the link has no caption the label must precede a section header)
make[1]: Leaving directory '/workspace/acrn-hypervisor/doc'
make[1]: *** [Makefile:57: html] Error 1
make: *** [Makefile:72: doc] Error 2
Build step 'Execute shell' marked build as failure

File: doc/release_notes_0.6.rst [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, and documentation.
```
Comment (gvancuts) [inline doc/release_notes_0.6.rst]: Add `tools` to this list

File: doc/release_notes_0.6.rst [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+https://projectacrn.github.io/0.6/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+ACRN v0.6 requires Clear Linux OS version 27600 or newer.  Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 0.6 new features
+************************
+
+**Support PREEMPT_RT-Linux for real-time UOS in ACRN**:
```
Comment (gvancuts) [inline doc/release_notes_0.6.rst]: I would like that we more clearly split the difference between the ACRN ability to run "Privileged VMs" and the "recipe" we have published to use a UOS kernel that includes the PREEMPT_RT patches. Both are linked but should be loosely coupled. In my opinion, this paragraph should emphasize the fact we are enabling "Privileged VMs" in ACRN, this is what the title should reflect. The paragraph can also mention that as a result, we have published a tutorial on how to also use the PREEMPT_RT patches in a UOS kernel.

**Comment by dbkinder** [comment]
I'm going to merge this so others can submit edits as more release information is obtained.

**Pull request merged**